worlds/stardew_valley/__init__.py

```diff
@@ -64,6 +64,12 @@ class StardewValleyWorld(World):
     item_name_to_id = {name: data.code for name, data in item_table.items()}
     location_name_to_id = {name: data.code for name, data in location_table.items()}
 
+    item_name_groups = {group.name.replace("_", " ").title() + (" Group" if group.name.replace("_", " ").title()
```

Comment:
Should this behavior of renaming some groups to "X Group" if they are named the same as an item, be also applied to locations?

Reply:
I don't believe it causes any collisions right now? unless those aren't checked for in unit tests?

Reply:
Agreed, but as a future-proofing type of thing, I think they should both have it